cli/pubsub/pubsub.go

```diff
@@ -81,6 +81,23 @@ func subscribeAction(c *cli.Context) error {
 		txn, _ := MakeUnsubscribeTransaction(myWallet, id, topic, nonce, txnFee)
 		buff, _ := txn.Marshal()
 		resp, err = client.Call(Address(), "sendrawtransaction", 0, map[string]interface{}{"tx": hex.EncodeToString(buff)})
+	case c.Bool("get"):
+		topic := c.String("topic")
+		if topic == "" {
+			fmt.Println("topic is required with [--topic]")
+			return nil
+		}
+
+		params := map[string]interface{}{"topic": topic}
+		subscriber := c.String("subscriber")
+		if subscriber == "" { // get list of topic
+			params["offset"] = c.Uint64("offset")
+			params["limit"] = c.Uint64("count")
```

Comment:
Suggest to rename `count` to `limit` for API consistency and naming convention

Reply:
Done.